README.md

```diff
@@ -50,14 +50,14 @@ Copy the template. Rename to include the date of intended publication in the fil
 
 ## Send the email
 
-**Create the campaign** using the [This Week in Tessel template](http://templates.mailchimp.com/getting-started/using-mailchimp/).
+**Create a new campaign**. Name the campaign something like "This Week in Tessel - Date". Set the subject to "This Week in Tessel: *Some content here...*"
```

Comment:
Campaign name should probably just be the same as the subject line. I assume it provides date metadata anyway?